### PR TITLE
[nodejs] update goldens

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: server/tests/test_data/nodejs_query/scatter/screenshot.json [hunk 1]
```diff
@@ -1,46 +1,3 @@
...
-          "url": "https://www.census.gov/programs-surveys/acs/data/data-via-ftp.html"
-        }
-      ],
-      "title": "Population Below Poverty Line in Counties of California (2021)",
-      "type": "MAP",
-      "unit": "",
-      "chartUrl": ""
-    }
-  ]
+  "charts": []
```
Comment (pradh) [inline server/tests/test_data/nodejs_query/scatter/screenshot.json:2]: Why don't we see the ranking chart that's 3rd in https://datacommons.org/explore/#q=obesity%20vs.%20poverty%20in%20counties%20of%20california ?

File: server/tests/test_data/nodejs_query/timeline_all/screenshot.json [hunk 2]
```diff
@@ -18,7 +18,8 @@
         "date": "2019",
         "value": 105819
       },
-      "chartUrl": ""
+      "chartUrl": "",
+      "dcUrl": "https://datacommons.org/explore#q=family%20earnings%20in%20north%20dakota"
```
Comment (pradh) [inline server/tests/test_data/nodejs_query/timeline_all/screenshot.json:22]: nit: we should have a TODO to switch this to bard.datacommons.org (in a couple of days)

**Review by pradh — APPROVED** [review]
Nice!

File: server/tests/test_data/nodejs_query/timeline_all/screenshot.json (hunk 2 above)
Comment (chejennifer) [reply server/tests/test_data/nodejs_query/timeline_all/screenshot.json:22]: will include this in followup PR that will also fix the ranking bug

**Pull request merged**